lib/shoryuken/extensions/active_job_adapter.rb

```diff
@@ -88,6 +79,13 @@ def perform(_sqs_msg, hash)
           Base.execute hash
         end
       end
+
+      MESSAGE_ATTRIBUTES = {
+        'shoryuken_class' => {
+          string_value: JobWrapper.to_s,
+          data_type: 'String'
+        }
+      }.freeze
```

Comment:
That's a good catch. constant makes sense.

Reply:
Thanks @phstc  -- We'll put together a better PR a bit later.  Turns out a frozen constant messes up our code downstream.